### PR TITLE
QPPCT-509: regard filename as pii

### PR DESCRIPTION
**Pull request by clydet**
### Information
- JIRA QPPCT-509

### Changes proposed in this PR:
- regard filename as PII

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Comment by sfradkin** [comment]
Just want to verify that a search was made in the codebase for anything filename related.

Also, if there is a way to perform an integration test to verify that the filename value is being encrypted, it would be good to have in here.

**Comment by saquino0827** [comment]
An integration test @sfradkin seems like a good idea for how serious the PII issue was. Are we adding that in this PR? Otherwise, I'll approve.

File: converter/src/main/java/gov/cms/qpp/conversion/Converter.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public Context getContext() {
 	 * @return status of conversion
 	 */
 	public JsonWrapper transform() {
-		DEV_LOG.info("Transform invoked with file {}", source.getName());
```
Comment (omnivion) [inline converter/src/main/java/gov/cms/qpp/conversion/Converter.java]: I think we should document why we're making this change in the code

**Comment by clydet** [comment]
@halprin true, there is an occurrence in command line, but that's not of concern.

**Comment by clydet** [comment]
@sfradkin not sure how feasible that will be. The file name will be encrypted at rest so before write and after read it will be decrypted.

**Pull request merged**